src/interpreters/mitie_interpreter.py

```diff
@@ -25,9 +34,12 @@ def get_entities(self, text):
                 expr = re.compile(_regex)
                 m = expr.search(text)
                 start, end = m.start(), m.end()
+                entity_value = text[start:end]
+                if entity_value in self.entity_synonyms:
```

Comment:
yes, I think it does makes sense. We should take care of that also during training